Docs/radium-as-submodule.md

```diff
@@ -0,0 +1,90 @@
+# How to use Radium-Engine as an external dynamic lib for your project
+
+## Fork pre made AppExample
+
+You can also fork https://github.com/STORM-IRIT/Radium-AppExample 
+It already contains Radium as submodule and a build-radium.sh script
+in the external directory.
+
+## git submodule, or just clone
+
+You can clone Radium, or add it as a submodule of your git project
+git submodule radium in my own project
+For instance Radium in ./myproject/external/Radium-Engine
```

Comment:
I don't understand this sentence!

Reply:
which part ?
You can clone Radium -> `git clone https://github.com/STORM-IRIT/Radium-Engine.git`
or add it as a submodule of your git project
-> `git submodule add  https://github.com/STORM-IRIT/Radium-Engine.git external/Radium-Engine`
line 12 could be removed ...
I will reformulate